src/Playwright/Transport/StdIOTransport.cs

```diff
@@ -155,58 +129,35 @@ private void Dispose(bool disposing)
 
         private async Task GetResponseAsync(CancellationToken token)
         {
-            try
-            {
-                var stream = _process.StandardOutput;
-                byte[] buffer = new byte[DefaultBufferSize];
+            var stream = _process.StandardOutput;
 
-                while (!token.IsCancellationRequested && !_process.HasExited)
-                {
+            while (!token.IsCancellationRequested && !_process.HasExited)
+            {
+                byte[] lengthBuffer = new byte[4];
```

Comment:
Would be nice if this could be reused (e.g. put outside of loop) and maybe clear it `lengthBuffer.AsSpan().Clear();` if necessary.